pro/src/app/App/analytics/firebase.ts

```diff
@@ -96,6 +96,20 @@ export const useFirebase = (consentedToFirebase: boolean) => {
   }, [selectedOffererId, isFirebaseInitialized])
 }
 
+export const useRemoteConfigParams = () => {
+  const [remoteConfigParams, setRemoteConfigParams] = useState<
+    Record<string, string>
+  >({})
+
+  useEffect(() => {
+    if (firebaseRemoteConfig) {
+      setRemoteConfigParams(getRemoteConfigParams())
+    }
+  }, [])
```

Comment:
J'ai l'impression qu'il manque ça, non ? Sinon, si remoteConfig n'est pas dispo, on ne renverra rien jusqu'au prochain appel, de `useRemoteConfigParams`

```suggestion
  }, [firebaseRemoteConfig])
```